code/go/pkg/validator/validator_test.go

```diff
@@ -173,6 +173,12 @@ func TestValidateFile(t *testing.T) {
 				"field streams.0.vars.3: Must not be present",
 			},
 		},
+		"bad_subobjects": {
+			"data_stream/rules/fields/base-fields.yml",
+			[]string{
+				`field 5.type: 5.type must be one of the following: "object"`,
```

Comment:
Looks like CI is failing because of this, the wrong field is in sixth position now.
```suggestion
				`field 6.type: 6.type must be one of the following: "object"`,
```